docs/bcfusion/validators/gov.md

```diff
@@ -0,0 +1,102 @@
+
+### Submit Proposal
+
+To submit a proposal, a staking credit holder needs to send a `propose` transaction to the BNB smart chain, specifying
+the following information:
+
+- **Proposer address**: The BEP20 address of the proposer, who initiates the proposal and pays the proposal fee.
+- **Targets**: The list of addresses of the contracts or accounts that the proposal wants to interact with.
+- **Values**: The list of amounts of BNB or other tokens that the proposal wants to transfer to the targets.
+- **Signatures**: The list of function signatures of the contracts that the proposal wants to call on the targets.
+- **Calldatas**: The list of encoded arguments of the functions that the proposal wants to call on the targets.
+- **Description**: The description of the proposal, which provides more details and rationale for the proposal.
+
+A delegator should delegate more the 200 staked BNB as the minimal requirement for submitting a proposal.
+The `propose` transaction will deduct the proposal fee (TODO: currently there is no such fee, we may add the fee) 
+from the proposer address and create a new proposal on the BNB
+smart chain. The proposal will have a unique proposal ID, and a proposal status of `Pending`. The proposal will then
+enter the voting period, which is the time window for the staking credit holders to cast their votes on the proposal.
+
+### Cast Vote
+
+To cast a vote on a proposal, a staking credit holder needs to send a `castVote` transaction to the BNB smart chain,
+specifying the following information:
+
+- **Voter address**: The BEP20 address of the voter, who casts the vote on the proposal. The voter address can be the
+  same as the staking credit holder address, or a different address that is delegated by the staking credit holder to
+  participate in governance on their behalf.
+- **Proposal ID**: The ID of the proposal, which identifies the proposal that the voter wants to vote on.
+- **Support**: The boolean value of the vote, which indicates the voter's preference on the proposal. `True` means that
+  the voter supports the proposal, and `False` means that the voter opposes the proposal.
+
+The `castVote` transaction will record the support value and the voting power of the voter on the BNB smart chain. The
+voting power is the amount of staking credit that the voter has at the time of the vote. The voting power can change due
+to staking operations, such as delegate, undelegate, or redelegate, but the support value will remain the same. The
+voter can change their support value at any time during the voting period, by sending another `castVote` transaction
+with a different support value.
+
+After submitting a proposal, the staking credit holders can cast their votes on the proposal until the voting period
+ends. The voting period is one of the BNB smart chain governance parameters, and it is currently set to 7 days.
+
```

Comment:
can we add a part about how we tally the votes, in what condition the proposal is approved?

Reply:
In `execution` part, there is some introduction of this topic.